Develop

stevefaulkner opened the pull request:
merge dev into main branch

Commit:
small change

https://github.com/ThePacielloGroup/CCA-Win/issues/24

Commit:
bug fix: access violation error

Commit:
fixed some issues

Commit:
bug fix: cca.ini not found error

Commit:
small bug fix

patrickhlauke commented on package/lang/Default.ini:
```diff
@@ -56,7 +56,7 @@ en_mail=sfaulkner@paciellogroup.com
 WATC=http://www.wat-c.org/
 abouttext=The Colour Contrast Analyser was developed by Jun in collaboration with Steve Faulkner.
-versiontext=Colour Contrast Analyser version 2.4
+versiontext=Colour Contrast Analyser version:
```
seems that this leaves it dangling as just "Colour Contrast Analyser version:" ?

jun325 replied:
In the latest version, the version number is acquired from the binary file.
![ccass](https://cloud.githubusercontent.com/assets/5038132/23248450/c318e768-f9e2-11e6-8504-70dc991dc3f5.JPG)

patrickhlauke replied:
Oh, excellent (need to make sure I check that before making a release, so I match the exact version from the code)

patrickhlauke commented on package/lang/Default.ini:
```diff
@@ -130,7 +130,7 @@ res_large_text_AA=Large text %s at Level AA
 
 lcr_ratio_is=The contrast ratio is: %s
-contrast_ratio=Contrast ratio: %s
+just_below=The contrast ratio is: just below %s
```
this will presumably need to be added (in translated form) to all other language versions?

jun325 replied:
Yes.

patrickhlauke replied:
Ok I'll see if I can get some kindly volunteers to help out with the remaining ones...

patrickhlauke committed:
Changes to language ini files

- version number now automatically added (no need to explicitly change
this each time)
- add text to "just below..." contrast ratio
- minor type fixes / spacing

patrickhlauke commented:
Closing this pull request, going to make a fresh one which includes the above changes to ini files
